src/devtools/client/debugger/src/reducers/ui.js

```diff
@@ -113,6 +119,10 @@ export function getPaneCollapse(state) {
   return state.ui.startPanelCollapsed;
 }
 
+export function getSourcesCollapse(state) {
```

Comment:
Small nit but can we change references to `getSourceCollapse` to `getSourcesCollapsed` for consistency?